client/active-game/active-game-manager.js

```diff
@@ -82,13 +96,32 @@ export default class ActiveGameManager extends EventEmitter {
   }
 
   setGameRoutes(gameId, routes) {
-    if (!this.activeGame || this.activeGame.id !== gameId) {
+    const current = this.activeGame
+    if (current && current.id !== gameId) {
       return
     }
 
-    this.activeGame.routes = routes
-    this.emit('gameCommand', gameId, 'routes', routes)
-    this.emit('gameCommand', gameId, 'setupGame', this.activeGame.config.setup)
+    if (current && current.config && routes) {
```

Comment:
Is there a place where `routes` would be undefined/null here? Feels kinda weird and maybe like we should fix that to pass an empty array/list if so, instead of doing this check

Reply:
No, `routes` should always be set to something here. This was just a case of defensive programming that's not needed; I'll remove it.